api/v1alpha1/clusterrequest_types.go

```diff
@@ -52,6 +52,14 @@ type ClusterRequestStatus struct {
 
 	// Contains JSON schema and cluster template validation details.
 	ClusterTemplateInputValidation ClusterTemplateInputValidation `json:"clusterTemplateInputValidation,omitempty"`
+	// RenderedTemplate represents whether the ClusterInstance template is rendered successfully
```

Comment:
We could consider having all the`RenderedTemplate*` status vars in a separate structure.